count_string.py

```diff
@@ -0,0 +1,12 @@
+def count_occurences_of_letter_in_string(word, letter_to_count):
+    """casefolding words and letters so
+    we can test for letter regardless of case."""
```

Comment:
Comments formatted like this are for a very specific purpose. They're called docstrings and are supposed to provide documentation about what the function does.

For example, this:

```python
def count_occurences_of_letter_in_string(word, letter_to_count):
    """Count occurrences of a letter on string, regardless of case."""
    …
```

Your comment explains code inside the function, so it's not an appropriate docstring. Your comment should be marked with `#` characters.

```python
def count_occurences_of_letter_in_string(word, letter_to_count):
    # casefolding words and letters so
    # we can test for letter regardless of case.
    …
```